maintenance_releases.es.Rmd

```diff
@@ -1,43 +1,44 @@
+-   Actualiza el archivo de noticias antes de cada envío a CRAN, con una sección con el nombre del paquete, la versión y la fecha de envío, como (como se ve en nuestro [archivo *NEWS* de ejemplo](#newstemplate)):
 
 ```
 foobar 0.2.0 (2016-04-01)
 =========================
 ```
 
-- Debajo de esa cabecera, pon las secciones que sean necesarias, incluyendo `NEW FEATURES`, `MINOR IMPROVEMENTS`, `BUG FIXES`, `DEPRECATED AND DEFUNCT`, `DOCUMENTATION FIXES` y cualquier epígrafe especial que agrupe un gran número de cambios. Debajo de cada encabezado, enumera los elementos según sea necesario (como se ve en nuestro ejemplo [archivo de noticias](#newstemplate)). Para cada elemento, proporciona
-  una descripción de la nueva característica, mejora, corrección de errores o función/característica obsoleta. Enlaza
-  a cualquier issue de GitHub relacionado como `(#12)`. El `(#12)` se resolverá en GitHub en Releases a un enlace a ese issue en el repo.
-- Después de haber añadido un `git tag` y lo hayas empujado a GitHub, añade las noticias de esa versión etiquetada a las notas de publicación de una versión en tu repo de GitHub con un título como `pkgname v0.1.0`. Consulta [Documentación de GitHub sobre la creación de una publicación](https://help.github.com/articles/creating-releases/).
-- Se escribirá sobre las nuevas versiones de CRAN [en nuestro boletín quincenal](https://news.ropensci.org/) pero ver [siguiente capítulo sobre marketing](#marketing) sobre cómo informar a más usuarios potenciales sobre el lanzamiento.
-- Para obtener más orientación sobre el archivo NEWS, te sugerimos que leas la [guía de estilo de tidyverse NEWS](https://style.tidyverse.org/news.html).
-
+-   Debajo de esa cabecera, agrega las secciones que sean necesarias, incluyendo "Nueva funcionalidad", "Mejoras menores", "Bugs resueltos", "Funciones obsoletas y caducas", "Mejoras en la documentación" y cualquier epígrafe especial que agrupe un gran número de cambios. Debajo de cada encabezado enumera los elementos según sea necesario (como se ve en nuestro [archivo *NEWS* de ejemplo](#newstemplate)). Para cada elemento, proporciona una descripción de la nueva característica, mejora, error corregido o función/característica obsoleta. Enlaza a cualquier *issue* de GitHub relacionado como `(#12)`. El `(#12)` se reemplazará en GitHub como un enlace a ese *issue* en el repo.
+-   Después de haber añadido un `git tag` y lo hayas enviado a GitHub, añade las noticias de esa versión etiquetada a las notas de publicación de una versión en tu repo de GitHub con un título como `pkgname v0.1.0`. Consulta [la documentación de GitHub sobre la creación de una publicación](https://help.github.com/articles/creating-releases/).
 
+-   Nuestro [boletín quincenal](https://news.ropensci.org/)] describirá los paquetes con nuevas versiones en CRAN. Ver el [siguiente capítulo sobre marketing](#marketing) sobre cómo informar a más usuarias/os potenciales sobre el lanzamiento.
```

Comment:
Is this “boletín quincenal” our newsletter? Because it is monthly now. Perhaps we can only said our newsletters without specify the time of publication? @maelle what do you think?

Reply:
good idea! would you have time to make a PR to the English file?